webapp/client/cypress/integration/personAHasDisability.spec.js

```diff
@@ -31,9 +31,12 @@ describe("PersonAHasDisability", () => {
       cy.get("#person_a_has_disabilityno").should("not.be.checked");
     });
 
-    it("Should link forward to telephone number page", () => {
+    it("Should link forward to person a requests fahrkostenpauschale", () => {
       cy.get("button[type=submit]").click();
-      cy.url().should("include", "/lotse/step/telephone_number");
+      cy.url().should(
+        "include",
+        "/lotse/step/person_a_requests_fahrkostenpauschale"
+      );
```

Comment:
When merging this probably has to be changed to the way it already is on main and the pauschbetrag functional tests have to include something like this test.